### PR TITLE
Migrate how to create a loyalty program using meta transactions from medium to docs

### LINKED ISSUE #435: Migrate How to create a Loyalty Program using Meta-transactions from Medium to docs
[Blog Link](https://developers.celo.org/how-to-create-a-loyalty-program-using-meta-transactions-686ae036b976)

### PR DESCRIPTION
**Pull request by therealharpaljadeja**
Closes #435

### CODE REVIEW

**Comment by therealharpaljadeja** [comment]
don't merge this PR until you merge this 

https://github.com/celo-org/DevRel/pull/160

**Review by joenyzio — APPROVED** [review]
LGTM!

**Pull request merged**
